src/Microsoft.AspNetCore.Antiforgery/IAntiforgery.cs

```diff
@@ -36,12 +36,13 @@ public interface IAntiforgery
         AntiforgeryTokenSet GetTokens(HttpContext httpContext);
 
         /// <summary>
-        /// Asynchronously returns a value indicating whether the request contains a valid antiforgery token.
+        /// Asynchronously returns a value indicating whether the request passes antiforgery validation. If the
+        /// request uses a safe HTTP method (GET, HEAD, OPTIONS, TRACE), the antiforgery token is not validated.
         /// </summary>
         /// <param name="httpContext">The <see cref="HttpContext"/> associated with the current request.</param>
         /// <returns>
-        /// A <see cref="Task{bool}"/> that, when completed, returns <c>true</c> if the request contains a
-        /// valid antiforgery token, otherwise returns <c>false</c>.
+        /// A <see cref="Task{bool}"/> that, when completed, returns <c>true</c> if the is valid, otherwise 
+        /// returns <c>false</c>.
```

Comment:
Hard to determine from these comments what the method will return if the request uses a safe HTTP method. Does "not validated" mean it returns `true` or `false`? Please clarify.